testutils/testutils.go

```diff
@@ -103,3 +111,30 @@ func (r RouterGroupMatcher) Matches(x interface{}) bool {
 func (r RouterGroupMatcher) String() string {
 	return fmt.Sprintf("router group's base path is %s", r.Path)
 }
+
+// ConnectToIntegrationTestDB waits for the integrations tests DB to become available
+// and returns a connection to the DB
+func ConnectToIntegrationTestDB(t *testing.T) *mongo.Database {
+	client, err := mongo.NewClient(options.Client().ApplyURI("mongodb://hs_auth:password123@localhost:8003/hs_auth"))
```

Comment:
Where are the username and password set for the mongodb? If it isn't too much trouble can they be moved into some constants?

Reply:
They are set in `docker/hs_auth_test/env_test`. I've moved them out to constants so it's a bit more clear where they are in the code. I don't want to fetch their values with an OS call because that would mean that they would need to be set on the host's environment and that can mess with the deployment scripts